serverless/src/utils/fetchEdlProfile.js

```diff
@@ -22,14 +20,29 @@ const fetchEdlProfile = async (headers) => {
   const clientToken = await fetchEdlClientToken()
 
   return fetch(`${host}/api/nams/edl_user`, {
-    body: `token=${token}`,
+    body: `token=${launchpadToken}`,
```

Comment:
Want to ensure that EDL is expecting an LP token here.

Reply:
It is their NAMS API, so yes